src/main/java/graphql/Directives.java

```diff
@@ -31,16 +31,25 @@
 @PublicApi
 public class Directives {
 
-    private static final String SPECIFIED_BY = "specifiedBy";
     private static final String DEPRECATED = "deprecated";
+    private static final String INCLUDE = "include";
+    private static final String SKIP = "skip";
+    private static final String SPECIFIED_BY = "specifiedBy";
     private static final String ONE_OF = "oneOf";
     private static final String DEFER = "defer";
 
-    public static final String NO_LONGER_SUPPORTED = "No longer supported";
     public static final DirectiveDefinition DEPRECATED_DIRECTIVE_DEFINITION;
+    public static final DirectiveDefinition INCLUDE_DIRECTIVE_DEFINITION;
+    public static final DirectiveDefinition SKIP_DIRECTIVE_DEFINITION;
     public static final DirectiveDefinition SPECIFIED_BY_DIRECTIVE_DEFINITION;
     @ExperimentalApi
     public static final DirectiveDefinition ONE_OF_DIRECTIVE_DEFINITION;
+    @ExperimentalApi
+    public static final DirectiveDefinition DEFER_DIRECTIVE_DEFINITION;
```

Comment:
> Do we need to declare the defer one as experimental?

@andimarek it is declared here, did you also want the annotation to appear elsewhere?

Elsewhere all the defer codepaths have been annotated with `@ExperimentalApi`

Reply:
perfect, didn't see that